README.md

```diff
@@ -42,19 +42,46 @@ Thus, `window.React` is then available to the Chrome React devtools extension.
 Alternately, you can set this in your config file:
 
 ```
+// webpack 1.x usage
 module: {
   loaders: [
     { test: require.resolve("react"), loader: "expose-loader?React" }
   ]
 }
+
+// webpack 2.x usage
```

Comment:
@wuxiandiejia Please separate into own code blocks and instead of a comment add **webpack v1**/**webpack v2** above each block, you can even omit the `webpack` imho, but that's up to your taste 😛